plugins/labhub.py

```diff
@@ -161,3 +161,41 @@ def unassign_cmd(self, msg, match):
                 return 'You are not an assignee on the issue.'
         except KeyError:
             return 'Repository doesn\'t exist.'
+
+    @re_botcmd(pattern=r'mark\s+(wip|pending)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               flags=re.IGNORECASE)
+    def mark_cmd(self, msg, match):
+        """Mark a given PR/MR with status labels."""  # Ignore QuotesBear
+        state, host, org, repo_name, xr, number = match.groups()
+
+        if host.lower() == 'github':
+            assert xr.lower() == 'pull'
+        elif host.lower() == 'gitlab':
+            assert xr.lower() == 'merge_requests'
+
+        try:
+            mr = self.REPOS[repo_name].get_mr(number)
+            current_labels = mr.labels
+            if state == 'wip':
+                pending_labels = ['process/pending_review',
+                                  'process/pending review']
+                for label in pending_labels:
+                    current_labels.remove(label)
```

Comment:
You can do it like `[current_labels.remove(label) for lable in pending_labels if label in current_labels]`